xbmc/guilib/GUIImage.dox

```diff
@@ -72,6 +72,19 @@ important, as `xml` tags are case-sensitive.
 | fadetime      | This specifies a crossfade time that will be used whenever the underlying <b>`<texture>`</b> filename changes. The previous image will be held until the new image is ready, and then they will be crossfaded. This is particularly useful for a large thumbnail image showing the focused item in a list, or for fanart or other large backdrops.
 | background    | For images inside a container, you can specify background="true" to load the textures in a background worker thread. [See here for additional information about background loading](http://kodi.wiki/view/Background_Image_Loader).
 
+For the tags "texture" and "bordertexture", the following attributes are available.
+
+| Attribute     | Description                                                   |
+|--------------:|:--------------------------------------------------------------|
+| border        | Used to specify a region of the texture to be considered a border that should not be scaled when the texture is scaled to fit a control's dimensions. The portion in the border region will remain unscaled. Particularly useful to achieve rounded corners that do not change size when a control is resized. Note that zoom animations and GUI rescaling will still affect the border region - it is just the scaling of the texture to the control size which is unaffected. Using border="5" will give a 5 pixel border all around the texture. You can specify each of the border amounts on each edge individually using border="left,top,right,bottom".
```

Comment:
I'm not sure if doxygen allows for this, but it would be nice to wrap lines at 80 chars. Something like:

```
| Attribute     | Description                                                   |
|--------------:|:--------------------------------------------------------------|
| border        | Used to specify a region of the texture to be considered a
|               | border that should not be scaled when the texture is scaled to
|               | fit a control's dimensions. The portion in the border region
|               | will remain unscaled. Particularly useful to achieve rounded
|               | corners that do not change size when a control is resized.
|               | Note that zoom animations and GUI rescaling will still affect
|               | the border region - it is just the scaling of the texture 
|               | the control size which is unaffected. Using border="5" will
|               | give a 5 pixel border all around the texture. You can specify
|               | each of the border amounts on each edge individually using
|               | border="left,top,right,bottom".
|               |
| infill        | Available if the "border" attribute is used. Hint for the GUI
|               | drawing routine if the center portion or just the outer frame
|               | should be drawn. Useful if the texture center is fully
|               | transparent. Accepts boolean values "true" (default) and
|               | "false".
```

Reply:
Is it possible? I have no idea how doxygen works.

Reply:
nop it doesn't work, will break the table into multiple rows:
![image](https://user-images.githubusercontent.com/7375276/170836758-0fac53a6-8e6f-4bbd-adaa-ad5e29ef991e.png)

It's fine as is